src/components/SearchFilter.vue

```diff
@@ -534,17 +549,18 @@ export default {
     additionalFieldSelected(queryable) {
       const operators = queryable.getOperators(this.cql);
       this.filters.push({
+        id: `${queryable.id}-${Date.now()}-${Math.random()}`, // Unique ID
```

Comment:
I think we do similar things in multiple cases. Maybe it's time to have a central instance that creates IDs in duch cases.